### PR TITLE
[JBPM-9723] Add integration tests for Kafka WIH async

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-9723](https://issues.redhat.com/browse/JBPM-9723)

**referenced Pull Requests**:

* https://github.com/kiegroup/jbpm-work-items/pull/192

### CODE REVIEW

**Review by afalhambra — APPROVED** [review]
Just a question, but it's a great job

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by gmunozfe** [comment]
jenkins do fdb

File: pom.xml [hunk 1]
```diff
@@ -101,6 +101,7 @@
     <version.guru.nidi>0.18.0</version.guru.nidi>
     <version.io.springfox>2.9.2</version.io.springfox>
     <version.io.smallrye.openapi.core>2.0.26</version.io.smallrye.openapi.core>
+    <version.io.strimzi.strimzi-test-container>0.22.1</version.io.strimzi.strimzi-test-container>
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply pom.xml]: It's a recent update that wasn't there when coding this, updated, thanks a lot @afalhambra !

**Review by afalhambra — APPROVED** [review]
Thanks, great job!

**Comment by gmunozfe** [comment]
Unrelated test failure,
this is ready to merge @mareknovotny

**Comment by mareknovotny** [comment]
the problem here is that the FDB was interrupted in jbpm-wb, so we don't know if it finishes only with 1 test failure or the other repos can fail too with more issues.

**Comment by gmunozfe** [comment]
ok, running it again
jenkins do fdb

**Comment by mareknovotny** [comment]
we also probably need to extend timeout value as the build ran 12 hours and is not at the latest repository in the list :(
fyi @mbiarnes

**Comment by gmunozfe** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
just for a record the FDB timeout was extended to 14 hours (840 mins) so hopefully it is enough

**Comment by gmunozfe** [comment]
jenkins do fdb

**Comment by gmunozfe** [comment]
Timeout has been exceeded, launching again
jenkins do fdb

**Comment by mareknovotny** [comment]
hmm, it would be nice to compare the whole run locally how long it take on your computer @gmunozfe 
I am a little suspicious about the duration of the CI build, If there is so long run we need to think about eliminating long running test/testsuites in PR checks

**Comment by gmunozfe** [comment]
@mareknovotny this time has finished in less than 9 hours, and failures seem unrelated, wdyt?

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by gmunozfe** [comment]
@mareknovotny finally green

**Pull request merged**